core/navigation.js

```diff
@@ -20,6 +20,8 @@
 
 goog.provide('Blockly.Navigation');
 
+var connection = null;
```

Comment:
Make this Blockly.Navigation.connection, just so you don't accidentally shadow this variable with a local named "connection"